client/payment-details/summary/index.tsx

```diff
@@ -107,9 +107,17 @@ const PaymentDetailsSummary = ( {
 		: placeholderValues;
 	const renderStorePrice =
 		charge.currency && balance.currency !== charge.currency;
+
+	// We should only fetch the authorization data if the payment is marked for manual capture
+	const shouldFetchAuthorization =
+		charge.amount !== charge.amount_captured &&
+		charge.amount_refunded === 0 &&
+		displayCaptureAuthorizationSection;
+
 	const { authorization } = useAuthorization(
```

Comment:
Instead of dispatching `shouldFetchAuthorization` into `useAuthorization`, can we initialize with null instead? Something like a ternary operator?

I don't see the point to modify `useAuthorization`,  as it violates the single responsibility approach.